polkadot/runtime/westend/src/genesis_config_presets.rs

```diff
@@ -170,7 +171,7 @@ fn westend_testnet_genesis(
 	const ENDOWMENT: u128 = 1_000_000 * WND;
 	const STASH: u128 = 100 * WND;
 
-	let config = RuntimeGenesisConfig {
+	build_struct_json_patch!(RuntimeGenesisConfig {
```

Comment:
@michalkucharczyk Do you plan to clean up all the other testnet runtimes here or follow up?

Reply:
I planned to do this in follow-up.

Reply:
@bkontur: https://github.com/paritytech/polkadot-sdk/pull/6349